resources/navigating_WAI/scope.md

```diff
@@ -13,26 +13,38 @@ https://github.com/w3c-cg/aar/issues/6
 
 ## Scope
 
-TBD
+Creating new tags for existing WAI pages for users to search for or filter on.  OR create a new resource with links to specific resources or sections of the WAI website based on need and usability level.   
```

Comment:
I think the resource should be the primary scope item. Maybe switch these around and put them in an ordered list?
1. Create a new resource with links to specific resources or sections of the WAI website based on need and usability level.
2. Create new tags for existing WAI pages for users to search for or filter on.